.jshintrc

```diff
@@ -0,0 +1 @@
+{ "esversion":6 }
```

Comment:
My sublime linter uses [JSHINT](http://jshint.com/), so this is just a config file for the project.